client/src/pages/Actions.ts

```diff
@@ -134,6 +152,98 @@ export const getOrders = async () => {
   return (await fetchJson("GET", {}, ORDER_LIST_API, true)) || [];
 };
 
+/**
+ * Retrieves the list of items in an order.
+ *
+ * @param {string} orderName - The Spot order name in the form of {merchants/*\/orders/*}.
+ * @returns {CartItem[]} contents - The items in the order.
+ */
+export const getOrderContents = async (orderName: string) => {
+  // parse merchant ID from order name
+  let merchantId = "";
+  const matches = orderName.match(/(?<=merchants\/)(.*?)(?=\/orders)/gi);
```

Comment:
is this parsed from the url? If so, possible to use a url parser ? if not, it might be good idea to add more comment here on the regexp